app/src/main/res/layout/layout_listitem.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="@color/colorAccent1"
+    android:padding="15dp"
+    android:id="@+id/parent_layout">
+
+    <androidx.cardview.widget.CardView
+        android:layout_width="match_parent"
+        app:cardBackgroundColor="#cdcdcd"
```

Comment:
put the colours hash code in the colour xml file and use that from there rather that using the hash code directly

Reply:
done